src/Servers/Kestrel/Transport.Quic/src/Internal/QuicConnectionListener.cs

```diff
@@ -35,8 +35,21 @@ public QuicConnectionListener(QuicTransportOptions options, IQuicTrace log, EndP
             _context = new QuicTransportContext(_log, options);
             var quicListenerOptions = new QuicListenerOptions();
 
+            var listenEndPoint = (IPEndPoint)endpoint;
```

Comment:
Lol, you half-fixed https://github.com/dotnet/aspnetcore/issues/35200. Want to add the real exception now or wait?

Reply:
Might as well.

Reply:
[`5de4cea` (#35258)](https://github.com/dotnet/aspnetcore/pull/35258/commits/5de4cea25039709024b3c0353a495702e5cc294b)